### PR TITLE
Feat maintenance worker env vars

### PR DESCRIPTION
**Pull request by christyjacob4**
## What does this PR do?
Added new environment variables that define the maximum duration for log retention 
## Test Plan
Tests are already present.

## Related PRs and Issues
#766 

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
Yes

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -35,4 +35,7 @@ _APP_FUNCTIONS_CPUS=1
 _APP_FUNCTIONS_MEMORY=128
 _APP_FUNCTIONS_MEMORY_SWAP=128
 _APP_MAINTENANCE_INTERVAL=86400
-_APP_USAGE_STATS=enabled
\ No newline at end of file
+_APP_MAINTENANCE_EXECUTION_LOG_RETENTION=1209600
```
Comment (eldadfux) [inline .env:38]: The Appwrite way is more like: `_APP_MAINTENANCE_RETENTION_EXECUTION`

Comment (eldadfux) [reply .env:38]: Same for all..

**Review by eldadfux — CHANGES_REQUESTED** [review]
Make sure to also sync compose.phtml with all the changes. This is the file used for installation.

**Comment by karbonnanroid** [comment]
b

File: .env (hunk 1 above)
Comment (hk4crprasad) [reply .env]: Yup

Comment (karbonnanroid) [reply .env]: b

**Pull request merged**